### PR TITLE
Sang da verify click on see more in other compact sedans section on review page after click on see more compact sedans to show more item

### PR DESCRIPTION
**Pull request by Daosang08**
- Delete SauceLab folder and component

- Delete old demo.java

- Delete old Apptest.java

- Add Folder CarResearchPages

- Add File ResearchNewAndUsedCarsPage.java

- Add File ReviewCarsPage.java

- Add Folder CarResearchPagesTests

- Add File ResearchNewAndUsedCarsPageTest.java

- Modified configs.properties

- Modified pom.xml

- Modified CustomKeyword

- Fix name of KbbBasePage

### CODE REVIEW

**Comment by namnguyen-itvn** [comment]
@cuong9pm please review this PR

**Comment by cuong9pm** [comment]
Review this Test Case : 

- Wrong Test Case Name
- Coding convention: not good at ReviewCarPage
- no Function Assert to verify TestCase
- Have not used DataDriven

File: demo/src/test/java/com/example/test/core/keywork/CustomKeyword.java [hunk 1]
```diff
@@ -1,151 +1,212 @@
...
+        }
+
+    }
+
+    // *wait without scroll */
+    public WebElement waitForElementDisplayedWithoutScroll(WebElement element) throws InterruptedException {
         try {
-            JavascriptExecutor js = (JavascriptExecutor) driver;
-            js.executeScript("arguments[0].scrollIntoView(true);", element);
             Thread.sleep(3000);
```
Comment (cuong9pm) [inline demo/src/test/java/com/example/test/core/keywork/CustomKeyword.java]: should remove thread.sleep and find the way to handle timeout

File: demo/src/test/java/com/example/test/core/keywork/CustomKeyword.java [hunk 2]
```diff
@@ -1,151 +1,212 @@
...
-    public WebElement waitForElementIsDisplayed(WebElement element){
+    public CustomKeyword scrollAndWaitToClick(WebElement element) throws InterruptedException {
         try {
-            return wait.until(ExpectedConditions.elementToBeClickable(element));
-        } catch (WebDriverException e) {
-            throw new WebDriverException("Element is not displayed ");
+            scrollToElemtnIntoView(element);
+            waitForElementDisplayed(element);
+            element.click();
+            Thread.sleep(4500);
```
Comment (cuong9pm) [inline demo/src/test/java/com/example/test/core/keywork/CustomKeyword.java]: should remove thread.sleep and find the way to handle timeout

File: demo/src/test/java/com/example/test/core/keywork/CustomKeyword.java [hunk 3]
```diff
@@ -1,151 +1,212 @@
...
-    public CustomKeyword click(WebElement element){
-        Actions actions = new Actions(this.driver);
-        actions.moveToElement(element).build().perform();
-        waitForElementIsDisplayed(element).click();
-        return new CustomKeyword(driver);
+    public void scrollByToClick(WebElement element) throws InterruptedException {
+        try {
+            JavascriptExecutor js = (JavascriptExecutor) driver;
+            js.executeScript("window.scrollBy(0,500)");
+            Thread.sleep(3000);
```
Comment (cuong9pm) [inline demo/src/test/java/com/example/test/core/keywork/CustomKeyword.java]: should remove thread.sleep and find the way to handle timeout

File: demo/src/test/java/com/example/test/core/keywork/CustomKeyword.java [hunk 4]
```diff
@@ -1,151 +1,212 @@
...
+            element.click();
+            Thread.sleep(3000);
```
Comment (cuong9pm) [inline demo/src/test/java/com/example/test/core/keywork/CustomKeyword.java]: should remove thread.sleep and find the way to handle timeout

File: demo/src/test/java/com/example/test/core/keywork/CustomKeyword.java [hunk 5]
```diff
@@ -1,151 +1,212 @@
...
+        } catch (WebDriverException ex) {
+            throw new WebDriverException("Element not availabe to input text!");
+        }
+    }
+
+    public void argumentsByToClick(WebElement element) throws InterruptedException {
+        try {
+            JavascriptExecutor jse = (JavascriptExecutor) driver;
+            jse.executeScript("arguments[0].click()", element);
+            Thread.sleep(3000);
```
Comment (cuong9pm) [inline demo/src/test/java/com/example/test/core/keywork/CustomKeyword.java]: should remove thread.sleep and find the way to handle timeout

File: demo/src/test/java/com/example/test/core/keywork/CustomKeyword.java [hunk 6]
```diff
@@ -1,151 +1,212 @@
...
+        } catch (WebDriverException ex) {
+            throw new WebDriverException("Element not availabe to input text!");
+        }
+    }
+
+    public void scrollBackToClick(WebElement element) throws InterruptedException {
+        try {
+            JavascriptExecutor js = (JavascriptExecutor) driver;
+            js.executeScript("window.scrollBy(0,0)");
+            Thread.sleep(3000);
```
Comment (cuong9pm) [inline demo/src/test/java/com/example/test/core/keywork/CustomKeyword.java]: should remove thread.sleep and find the way to handle timeout

File: demo/src/test/java/com/example/test/core/keywork/CustomKeyword.java [hunk 7]
```diff
@@ -1,151 +1,212 @@
...
+            element.click();
+            Thread.sleep(3000);
```
Comment (cuong9pm) [inline demo/src/test/java/com/example/test/core/keywork/CustomKeyword.java]: should remove thread.sleep and find the way to handle timeout

File: demo/src/test/java/com/example/test/core/keywork/CustomKeyword.java [hunk 8]
```diff
@@ -1,151 +1,212 @@
...
+            throw new WebDriverException("Element not availabe to input text!");
+        }
+    }
+
+    public void selectElement(WebElement element, String text) throws InterruptedException {
+        try {
+            waitForElementDisplayed(element);
+            Select select6 = new Select(element);
+            select6.selectByVisibleText(text);
+            Thread.sleep(3000);
```
Comment (cuong9pm) [inline demo/src/test/java/com/example/test/core/keywork/CustomKeyword.java]: should remove thread.sleep and find the way to handle timeout

**Review by cuong9pm — COMMENTED** [review]
Please make your PR have change as the comment inline

**Pull request closed without merging**